specification/2.0/README.md

```diff
@@ -265,7 +265,7 @@ The [node transformations](#transformations) and [animation channel paths](#anim
 * rotation: A quaternion (x, y, z, w), where w is the scalar
 * scale: A 3D vector containing the scaling factors along the x, y and z axes
 
-
+RGB color values use sRGB (Rec. 709) color primaries.
```

Comment:
I don't know much about this, but from just some slight reading on it, I don't know if it's clear to me why Rec. 709 is here. Can we maybe add some links or maybe even the equation for converting to linear?

Reply:
sRGB and Rec. 709 use the same primaries.

Reply:
Okay, now I'm a bit confused what this statement means. Why mention sRGB or Rec. 709? Doesn't RGB already mean it's red, green, and blue and not some other primaries?

Reply:
Color primaries define what "red", "green", and "blue" mean — that's why this statement is in Units section.

Reply:
Yep.  Monitor/TV/printer folks have standards for reproducing these colors, and I'm fine with glTF making a casual external reference to that here.

Reply:
Ok, I must be missing something. Is there some article I can read to understand this more? My searches haven't come up with anything useful.

Reply:
I don't have any good links for this, but I'll try to offer some context here.  As Alexey points out, this particular sentence doesn't have anything to do with the transfer function.  If I understand correctly, it's talking about color reproductions, typically after the end of the graphics pipeline.  When an RGB image without a colorspace calls for color [1.0, 0.0, 0.0], we know it's asking for full red, but what does "full red" even mean?  Different monitors can have different capabilities as to the full gamut of colors they are able to express in hardware, and [1.0, 0.0, 0.0] may not map to exactly the same shade of red on every available monitor.  When sRGB calls for full red, they offer a precise definition of that red primary, and non-sRGB monitors should adapt the color as needed, to the best of the hardware capability.  Nailing down the exact primary colors like this means that we expect glTF to reproduce its colors the same way on a variety of different display technologies.

Reply:
Thanks. Alexey also gave me a few pointers which helps me understand what this statement means. I'm not sure whether people reading this spec will understand it. Perhaps it's just me.

Regardless though, what do glTF implementations do with this information?

Reply:
> what do glTF implementations do with this information?

Exporters: convert texels to sRGB (or warn when it's impossible) when authoring was done in something else (e.g. if original textures were for some reason authored using AdobeRGB).

Importers: ensure proper look on non-sRGB displays (e.g., with wide color gamut support).

Reply:
I agree with Gary that this statement is probably not going to be well understood by many people reading the spec. We could add a link to the Wikipedia page for Rec 709 and, ideally, add a few sentences to help demystify why we're talking about "primaries" and "transfer functions".
I don't think we should assume that the only people implementing exporters/importers are graphics engineers.

Reply:
ILM published an introduction to color management here - https://www.openexr.com/documentation/OpenEXRColorManagement.pdf - it goes beyond what gltf needs, but it explains all the foundational concepts from photography to display and how they relate to reproducing pixel values.

Reply:
The formal standards in this area are not very beginner-friendly.
- [sRGB definitions](http://www.color.org/sRGB.pdf)
- [ITU-R BT.709-6](https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.709-6-201506-I!!PDF-E.pdf)

I'd point to classic Poynton's FAQs, although they do not mention sRGB:
- https://poynton.ca/PDFs/GammaFAQ.pdf
- https://poynton.ca/PDFs/ColorFAQ.pdf

Another good guide for engine developers is NVIDIA's paper by Evan Hart: [UHD Color for Games](https://developer.nvidia.com/sites/default/files/akamai/gameworks/hdr/UHDColorForGames.pdf).

Reply:
I'm not sure the glTF spec should be attempting to educate beginners on this complex topic.  I like the PR the way it is now.  It references the relevant specifications, such that experts understand that we're following the most common standard in that area, and if a beginner needs to dig into that, they can go research that themselves.